x-pack/plugins/security_solution/public/common/hooks/use_url_state.ts

```diff
@@ -16,6 +15,8 @@ import { useQueryTimelineByIdOnUrlChange } from './timeline/use_query_timeline_b
 import { useInitFlyoutFromUrlParam } from './flyout/use_init_flyout_url_param';
 import { useSyncFlyoutUrlParam } from './flyout/use_sync_flyout_url_param';
 
+export const EXPANDABLE_FLYOUT_URL_KEY = 'eventFlyout' as const;
```

Comment:
maybe `detailsFlyout` is more suitable?

Reply:
yes, I'll leave that to the next PR though, this one is just about adding support to multiple flyouts